File: generated-sources/scalaz/mojang-authentication/src/main/scala/com/github/asyncmc/mojang/authentication/scalaz/model/UsernamePassword.scala
package com.github.asyncmc.mojang.authentication.scalaz.model

import argonaut._
import argonaut.EncodeJson._
import argonaut.DecodeJson._

import org.http4s.{EntityDecoder, EntityEncoder}
import org.http4s.argonaut._
import org.joda.time.DateTime
import UsernamePassword._

case class UsernamePassword (
  /* The Mojang account e-mail or username. Never store it. */
  username: String,
/* The Mojang account password, never store it. */
  password: String)

object UsernamePassword {
  import DateTimeCodecs._

  implicit val UsernamePasswordCodecJson: CodecJson[UsernamePassword] = CodecJson.derive[UsernamePassword]
  implicit val UsernamePasswordDecoder: EntityDecoder[UsernamePassword] = jsonOf[UsernamePassword]
  implicit val UsernamePasswordEncoder: EntityEncoder[UsernamePassword] = jsonEncoderOf[UsernamePassword]
}

File: generated-sources/php-ze-ph/mojang-authentication/src/App/com/github/asyncmc/mojang/authentication/php/ze/ph/model/UsernamePassword.php
<?php

namespace com.github.asyncmc.mojang.authentication.php.ze.ph.server\com.github.asyncmc.mojang.authentication.php.ze.ph.model;

use Articus\DataTransfer\Annotation as DTA;

/**
 */
class UsernamePassword
{
    /**
     * The Mojang account e-mail or username. Never store it.
     * @DTA\Data(field="username")
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $username;
    /**
     * The Mojang account password, never store it.
     * @DTA\Data(field="password")
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $password;
}
